Common/ShellFileItem.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Files.Common
+{
+    public class ShellFileItem
+    {
+        public bool IsFolder;
+        public string RecyclePath;
+        public string FileName;
+        public string FilePath;
+        public DateTime RecycleDate;
+        public string FileSize;
+        public ulong FileSizeBytes;
+        public string FileType;
+
+        public ShellFileItem()
+        {
+
+        }
```

Comment:
Is this necessary?

Reply:
I think so. I'm using Json.Net to serialize/deserialize ShellFileItem objects and as far as I know Json.Net need a parameterless constructor to to this.